spec/models/merchant_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Merchant, type: :model do
+  let(:empty_merchant) do
```

Comment:
very smart to create one empty merchant to use in all tests for presence validations :+1: